src/base/bittorrent/session.cpp

```diff
@@ -1917,7 +1917,7 @@ void Session::bottomTorrentsQueuePos(const QVector<InfoHash> &hashes)
     using ElementType = std::pair<int, TorrentHandle *>;
     std::priority_queue<ElementType
         , std::vector<ElementType>
-        , std::greater<ElementType>> torrentQueue;
+        , std::greater<>> torrentQueue;
```

Comment:
Why bother omitting it? Just keep the original, there is nothing wrong with it.

Reply:
The reasoning is: https://clang.llvm.org/extra/clang-tidy/checks/modernize-use-transparent-functors.html

But if you disagree and insist otherwise, I'll undo the change.

Reply:
I would still prefer the original one. You might able to use transparent functors in new code, but changing existing code, I wouldn't bother doing that.

Reply:
Well a cleanup is about simplifying existing code, but ok, I'll change it back.